ui/tests/acceptance/sync/secrets/destination-test.js

```diff
@@ -35,7 +35,7 @@ module('Acceptance | sync | destination', function (hooks) {
   });
 
   test('it should transition to correct routes when performing actions', async function (assert) {
-    await click(ts.navLink('Secrets Sync'));
+    await visit('/vault/sync/secrets/overview');
```

Comment:
Locally these would pass, but on GH they would not. This test is not testing sidebar nav things, so I used visit instead of a click on the navbar.

Reply:
FWIW - Navigating from the sidebar is good to incorporate into acceptance tests as it's how we've found transition bugs in the past from ember/ember data upgrades, etc. I'm not saying to revert this, but just wanted to mention. 

I wonder why these would fail on github 🤔  Maybe the test context there is not set up the same as locally so the new logic hiding the sidenav links isn't working properly

Reply:
Maybe it's failing because CI is running against a CE binary -- maybe check the mirage scenario and make sure it's returning enterprise seal status?

Reply:
Good idea!